### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by cristinabuenahora**
## Change description

Screenshot 2023-03-28 at 7 20 30 PM
Screenshot 2023-03-28 at 7 20 56 PM

## Type of change

- [ ] Bug fix (fixes an issue)
- [ ] New feature (adds functionality)
- [ ] Improvement (improves codebase without changing functionality)

## Related issues

> Fix [#1]()

### Development

- [ ] Lint rules pass locally
- [ ] Application changes have been tested thoroughly
- [ ] Automated tests covering modified code pass

### Security

- [ ] Security impact of change has been considered
- [ ] Code follows company security practices and guidelines

### Code review

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached as necessary
- [ ] "Ready for review" label attached and reviewers assigned
- [ ] Changes have been reviewed by at least one other contributor
- [ ] Pull request linked to task tracker where applicable

### CODE REVIEW

**Review by gsdatta — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,38 +1,36 @@
...
+visibility into their services and deliver high quality software.
+
+We’ve released our Scorecards product as a Backstage plugin. [Scorecards](https://www.cortex.io/products/scorecard) 
+allow your team to define standards like production readiness and development quality, 
+and enforce them without building scripts and maintaining spreadsheets.
+- **One-click integration with third-party tools**: Scorecards fetch data automatically from your integrations without manual work, letting you easily enforce standards across all your tools.
+- **The flexibility to meet your organization’s needs**: Our robust APIs make it easy to use data from custom sources in your Scorecards. Cortex Query Language (CQL) enables you to create complex rules that can compare data across multiple sources or write expressive logical statements.
+- **Enable leaders to make informed decisions**: Historical data and organizational summaries give leadership deep visibility into progress, bottlenecks, and areas of risk.
+- **Drive organizational progress with ease using Initiatives**: Within any Scorecard, assign owners and due dates to drive any best-practice, platform migration, and audit need.
```
Comment (FoodProduct) [inline README.md:12]:
```suggestion
- **Drive organizational progress with ease using Initiatives**: Within any Scorecard, assign owners and due dates to drive any best-practice, platform migration, and audit needs.
```

File: README.md [hunk 2]
```diff
@@ -1,38 +1,36 @@
...
+
+The plugin automatically ingests your Backstage components, and then lets you easily define
```
Comment (FoodProduct) [inline README.md:14]:
```suggestion
The plugin automatically ingests your Backstage components and then lets you easily define 
```

File: README.md [hunk 3]
```diff
@@ -1,38 +1,36 @@
...
+- [DORA metrics](https://www.cortex.io/post/building-a-dora-metrics-scorecard)
 
 ![plugin1](./docs/screen2.png?raw=true)
 ![plugin2](./docs/screen3.png?raw=true)
 
-Cortex can even message service owners with weekly reports over Slack or email, showing changes in the
-quality of the service. We also generate reports showing how different teams are
-performing in their Scorecards.
+Cortex creates personalized action items for service owners. These can be found in the Backstage UI, 
+or received through notifications via slack and email.
```
Comment (FoodProduct) [inline README.md:31]:
```suggestion
or received through notifications via Slack and email.
```

File: README.md [hunk 4]
```diff
@@ -1,38 +1,36 @@
...
+- Platform and package migrations
+- [DORA metrics](https://www.cortex.io/post/building-a-dora-metrics-scorecard)
 
 ![plugin1](./docs/screen2.png?raw=true)
 ![plugin2](./docs/screen3.png?raw=true)
 
-Cortex can even message service owners with weekly reports over Slack or email, showing changes in the
-quality of the service. We also generate reports showing how different teams are
-performing in their Scorecards.
+Cortex creates personalized action items for service owners. These can be found in the Backstage UI,
```
Comment (FoodProduct) [inline README.md:30]:
```suggestion
Cortex creates personalized action items for service owners. These can be found in the Backstage UI 
```

**Review by FoodProduct — COMMENTED** [review]
👏

**Review by FoodProduct — APPROVED** [review]

**Pull request merged**